SnowSchoolers/index.ios.js

```diff
@@ -1,9 +1,3 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
```

Comment:
This should have been replaced with "SnowSchoolers" and the link to the repository, instead of being completely wiped out.